libs/components/molecules/src/lib/header/header.component.html

```diff
@@ -0,0 +1,7 @@
+<a class="logo-link" [routerLink]="['/']"><img class="logo-image" [src]="logo" alt="" /></a>
+<p class="header-text-1">Human Reference Atlas</p>
```

Comment:
Should be part of the link
header-text-1 -> product-title
Use h1, h2, or whatever fits